x/ibc/04-channel/exported/exported.go

```diff
@@ -21,7 +21,8 @@ type CounterpartyI interface {
 // PacketI defines the standard interface for IBC packets
 type PacketI interface {
 	GetSequence() uint64
-	GetTimeoutHeight() uint64
+	GetTimeoutEpoch() uint64
```

Comment:
These needed to be split out into separate functions since `02-client/exported` imports `04-client/exported`, so I cannot return `clientexported.Height` here directly

NOTE: `02-client/types` imports `02-client/exported` so returning `clienttypes.Height` doesn't solve the problem

Reply:
I think we should move all these `exported.go` to the top level of IBC. If `exported.go` is causing code changes, we should try to fix the `exported.go` issue and not change the code. 

I'd only include the `core` IBC `exported.go` files as outlined by the code restructuring issue. They should all be contained in `x/ibc/exported/` and in the future `x/ibc/core/exported`. Each ICS module should have its own file `exported/client.go`, `exported/connection.go` etc